python/test/test_convert_graph.py

```diff
@@ -23,8 +24,8 @@ def test_adjacency_matrix():
     assert list(g.get_edge_property("weight").to_numpy()) == [1, 2, 3]
 
 
-def test_trivial_arrays():
-    g = from_edge_list_arrays(np.array([0, 1, 10]), np.array([1, 2, 0]))
+def test_trivial_arrays_unsorted():
+    g = from_edge_list_arrays(np.array([0, 10, 1]), np.array([1, 0, 2]))
```

Comment:
Shouldn't this also output range for 11 nodes? There will be 11 nodes in the graph, isn't it?

Reply:
There will be 11. And that's what's (implicitly) asserted by the assert below, while it's also checking that the edge ranges are correct.

I don't see any place where this test implies anything other than 11 nodes.

Reply:
Yes, I missed that. My bad.